rental.rb

```diff
@@ -0,0 +1,11 @@
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    book.rental << self
+    @person = person
+    person.rental << self
```

Comment:
- [ ] **[OPTIONAL]** You have `rentals` in your person class, not `rental`. You should consider fixing this.

  ```
  @person = person
  person.rentals << self
  ```